metricbeat/module/jolokia/jmx/data_test.go

```diff
@@ -222,6 +222,42 @@ func TestEventGroupingMapperGetRequest(t *testing.T) {
 	assert.ElementsMatch(t, expected, events)
 }
 
+// TestEventGroupingMapperGetRequestUptime tests responses which are returned
+// from a Jolokia GET request and only has one uptime runtime value.
+func TestEventGroupingMapperGetRequestUptime(t *testing.T) {
+	absPath, err := filepath.Abs("./_meta/test")
+
+	assert.NotNil(t, absPath)
+	assert.Nil(t, err)
```

Comment:
You can use `require` (`github.com/stretchr/testify/require`) instead of `assert` to fail fast in this kind of checks.

Reply:
Good to know! I always uses assert.NoError. I will change it to require.NoError then. Thanks!